Standard.AI.Data.EntityIntelligence/Services/Foundations/Datas/DataService.Exceptions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using Standard.AI.Data.EntityIntelligence.Models.Datas.Services;
+using Standard.AI.Data.EntityIntelligence.Models.Foundations.Datas.Exceptions;
+
+namespace Standard.AI.Data.EntityIntelligence.Services.Foundations.Datas
+{
+    internal partial class DataService : IDataService
+    {
+        private delegate ValueTask<IEnumerable<TableMetadata>> ReturningTableMetadatasFunction();
+
+        private static async ValueTask<IEnumerable<TableMetadata>> TryCatch(ReturningTableMetadatasFunction returningTableMetadatasFunction)
```

Comment:
Exceeds 120 char rule, suggest breaking to new line.


```suggestion
        private static async ValueTask<IEnumerable<TableMetadata>> TryCatch(
            ReturningTableMetadatasFunction returningTableMetadatasFunction)
```